deployments/k3d/Makefile

```diff
@@ -28,23 +33,26 @@ create-k3d: build-dev-k3d-image
 build-workload-images:
 	docker buildx build --platform=wasi/wasm --load -t $(TEST_IMG_NAME_SPIN) ../../images/spin
 	docker buildx build --platform=wasi/wasm --load -t $(TEST_IMG_NAME_SLIGHT) ../../images/slight
+	docker buildx build --platform=wasi/wasm --load -t $(TEST_IMG_NAME_WWS) ../../images/wws
 
 load-workload-images: build-workload-images
 	k3d image load $(TEST_IMG_NAME_SPIN)
 	k3d image load $(TEST_IMG_NAME_SLIGHT)
+	k3d image load $(TEST_IMG_NAME_WWS)
 
 up: create-k3d load-workload-images
 	kubectl label nodes k3d-k3s-default-agent-0 spin-enabled=true slight-enabled=true
```

Comment:
please add lable for wws-enabled=true

Reply:
Missed this one too! Thank you for catching it